_includes/venue.html

```diff
@@ -2,34 +2,46 @@
 <section id="venue">
     <div class="content-wrapper">
         <div class="col-lg-10 col-lg-offset-1">
+
             <h3>{{ site.venueBlockTitle }}</h3>
+
+            <div class="col-lg-6 col-lg-offset-3" style="margin-bottom: 60px; padding: 30px; border-style: solid; border-width: 2px; border-color: fff;">
+              <div style="font-size: 24px; line-height: 200%">
+
+                <p style="font-size: 24px; line-height: 200%">We believe that the venue page of this brainhack should reflect the beauty and the diversity of places and people that can be found on our planet.</p>
+                <p style="font-size: 24px; line-height: 200%">We would very much like for every brainhack attendee to add to this page a picture related to:</p>
+
+                <ul style="text-align: left">
+                  <li style="margin-left: 30px;">their home town or home country,</li>
+                  <li style="margin-left: 30px;">or the town or country they now live in,</li>
+                  <li style="margin-left: 30px;">or a place on this planet dear to their heart.</li>
+                </ul>
+
+                <p style="font-size: 24px; line-height: 200%">
+                  To do that you can follow the instruction in this <a href="#">issue</a> to guide you on how to make a pull request to this repository to add your image.</p>
```

Comment:
- The issue link was not working on my local version (bringing back to the issue page).